app/src/main/java/org/coepi/android/ui/symptoms/SymptomsViewModel.kt

```diff
@@ -60,8 +73,35 @@ class SymptomsViewModel(private val symptomRepo: SymptomRepo) : ViewModel() {
     fun onChecked(symptom: SymptomViewData) {
         checkedSymptomTrigger.onNext(symptom)
     }
+    fun doSubmitSymptms(){
+        log.i("About to send sypmtom:");
+        if( realmprovider!= null ){
+            var keys: String = "will be put by doPostSymptom using last 3 keys generated";
+            var report = "COVID 19 confirmed"
+            var id = 1;
+            var symptomReport = SymptomReport(id, report,keys);
```

Comment:
Don't use semicolons in Kotlin.
Use always `val` instead of `var`, unless you plan to mutate it.

Reply:
changed